4.1/start.sh

```diff
@@ -29,6 +35,20 @@ cd ${PROJECT_DIR} && sudo -E -u nominatim nominatim refresh --website --function
 
 service apache2 start
 
+# start continous replication process
+if [ "$REPLICATION_URL" != "" ] && [ "$FREEZE" != "true" ]; then
+  echo "starting replication"
+  # run init in case replication settings changed
+  sudo -u nominatim nominatim replication --project-dir /nominatim --init
+  sudo -u nominatim nominatim replication --project-dir /nominatim &> /var/log/replication.log &
+  replicationpid=${!}
+fi
+
 # fork a process and wait for it
-tail -f /var/log/postgresql/postgresql-14-main.log &
+tail -Fv /var/log/postgresql/postgresql-14-main.log /var/log/apache2/access.log /var/log/apache2/error.log /var/log/replication.log &
+tailpid=${!}
+
+echo "Warm database caches for search and reverse queries"
+sudo -E -u nominatim nominatim admin --warm > /dev/null
```

Comment:
again, we already do that in the init:

https://github.com/mediagis/nominatim-docker/blob/322deccac655584add9c4db08985323f506ee30f/4.1/init.sh#L97

is it advised to run the command after every restart of the database?

Reply:
the purpose of the warming is to put some data in the cache in RAM. After a reboot it makes sense to warm the cache.

Reply:
It runs 1000 queries against the database which usually gives the operating system and database time to put some indices into memory rather than reading from disk.

Reply:
How long does the warming take?

Reply:
On my server with the entire planet imported warming took 10min to finish. It runs in the background while the service is already available.

Reply:
That's ok for me.

Reply:
Ok, but if we run the warming procedure on every startup, we can remove it from the `init.sh` (https://github.com/mediagis/nominatim-docker/blob/322deccac655584add9c4db08985323f506ee30f/4.1/init.sh#L97)